snafu/benchmarks/nighthawk/Dockerfile

```diff
@@ -0,0 +1,11 @@
+ARG BASE_IMAGE=quay.io/cloud-bulldozer/nighthawk-base:latest
```

Comment:
Please include if you have a Dockerfile for this base image.

Reply:
It's just a copy of binaries from my local pod file system to image. So I did not include it. But the image is present in the cloud-bulldozer already. For more info on how those binaries are generated: https://github.com/envoyproxy/nighthawk

Reply:
The initial concern here is that we have no idea what version of nighthawk is being run and no "plan" in place on how to keep it updated as appropriate.